tests/screen-objects/hiltonMobileAppPage.js

```diff
@@ -0,0 +1,31 @@
+import BasePage from './basePage'
+
+class HiltonMobileAppPage extends BasePage {
+
+	constructor() {
+		super();
+	}
+
+	open() {
+		//do the operations that will open the hilton mobile app
+	}
+
+	isLoaded() {
```

Comment:
I would place the `isLoaded` in a base page, because you need it in each page, maybe check this as an example

- **Basepage:** https://github.com/saucelabs/sample-app-mobile/blob/master/tests/e2e/screenObjects/base.js#L15
- **usage of base page:** https://github.com/saucelabs/sample-app-mobile/blob/master/tests/e2e/screenObjects/login.js#L7